.github/workflows/crowdin-ci.yml

```diff
@@ -0,0 +1,68 @@
+name: Crowdin CI
+
+on:
+  schedule:
+    - cron: "20 4 1 * *" # Runs at 4:20 AM on the first day of every month
+  workflow_dispatch: # Can be dispatched manually
+
+jobs:
+  create_approved_language_bucket_prs:
+    runs-on: ubuntu-latest
+    env:
+      CROWDIN_API_KEY: ${{ secrets.CROWDIN_API_KEY }}
+      CROWDIN_PROJECT_ID: ${{ secrets.CROWDIN_PROJECT_ID }}
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
By providing this token, the GitHub CLI will automatically authenticate; have removed the `gh auth` step.